app/frontend/src/components/designer/FormSettings.vue

```diff
@@ -160,7 +198,7 @@
             :disabled="userType === ID_MODE.PUBLIC"
           >
             <template #label>
-              <span>
+              <span :class="{ 'mr-2': isRTL }">
                 Allow <strong> multiple draft</strong> upload
```

Comment:
Looks like we're missing all the translations for "Allow **multiple draft** upload". Should we fix in this PR or do it in another one?

Reply:
I will fix it